scrapy/commands/parse.py

```diff
@@ -7,7 +10,9 @@
 from scrapy.utils.conf import arglist_to_dict
 from scrapy.utils.spider import iterate_spider_output, spidercls_for_request
 from scrapy.exceptions import UsageError
-from scrapy import log
+
+logger = logging.getLogger('scrapy')
```

Comment:
What do you think about using `logging.getLogger(__name__)`? This way users will be able to adjust logging output of individual modules and subpackages. Stdlib logging supports loggers hierarchy: if you add a formatter and set a log level for 'scrapy' logger, this formatter and log level will be used for 'scrapy.commands.parse' as well, unless you override it.